en/docs/deploy/change-the-hostname.md

```diff
@@ -174,4 +174,20 @@ you can make sure that the existing encrypted data is not affected.
     data encryption, signing, etc.
     
 
-Once this is done, you need to change all `localhost` references.  
+Once this is done, you need to change all `localhost` references. The following points would be the main places where 
+the references should be changed.
+
+- The configurations in the `<IS_HOME/repository/conf/deployment.toml` file that references localhost.
+- The entity ID that is available in the resident identity provider configurations.
+- The callback URLs of the federated identity providers.
+- The callback URLs of the default system applications such as My Account and Console.
+
+!!! note
+
+    To edit the default system applications such as My Account and Console, the following configuration should be added
```

Comment:
What are we editing in the default system applications here?
Also, can you explain how this is relevant to changing the hostname?

@deshankoswatte

Reply:
These changes are added with https://github.com/wso2/docs-is/pull/3071/commits/5cd0ac947cc297f52fb1c3104382b1610e8e65be.